### DISCUSSION TITLE
issues：Doris be.conf 配置问题！

### DISCUSSION THREAD

**Discussion by LiHang-CN** [apache/doris#6959] [Show and tell]
BE节点：be.cconf
storage_root_path = /XX/XX/XX.HDD,300
或
storage_root_path = /XX/XX/XX,medium:hdd,capacity:300
配置无效
show proc '/backends';
查看到的 仍然是 BE节点整个磁盘大小

有可能是BUG

**Comment by hf200012** [comment]
Please refer to issues for this issue

**Reply by LiHang-CN** [reply]
Thank you. It's settled

**Comment by EmmyMiao87** [comment]
What's your version of Doris?

**Reply by LiHang-CN** [reply]
Thank you. It's settled
